ident/admin/index.php

```diff
@@ -140,8 +140,11 @@ function openHeadingAdmin(){
 											<ul>
 												<?php
 												$charList = $charArr[$hid];
-												foreach($charList as $cid => $charName){
-													echo '<li><a href="chardetails.php?cid=' . htmlspecialchars($cid, HTML_SPECIAL_CHARS_FLAGS) . '">' . htmlspecialchars($charName, HTML_SPECIAL_CHARS_FLAGS) . '</a></li>';
+												if (isset($cid) && isset($charName))
```

Comment:
I think that these two variables don't get defined until the for loop below, meaning that these would always evaluate to `false`. Am I mistaken?